src/main/java/greencity/config/SecurityConfig.java

```diff
@@ -47,10 +47,16 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests()
                 .antMatchers("/ownSecurity/**")
                 .permitAll()
+                .antMatchers("/place/getListPlaceLocationByMapsBounds/**")
+                .permitAll()
                 .antMatchers("/place/propose/**")
                 .hasAnyRole("USER", "ADMIN", "MODERATOR")
                 .antMatchers("/place/{status}")
                 .hasAnyRole("USER", "ADMIN", "MODERATOR")
+                .antMatchers("/place/status**")
+                .hasAnyRole("ADMIN", "MODERATOR")
+                .antMatchers("/user/role/**")
+                .hasAnyRole("USER", "ADMIN", "MODERATOR")
```

Comment:
You can  replace **hasAnyRole()** , which contains all ur roles with **permitAll()** 
P.S. optional